lib/matplotlib/axes/_axes.py

```diff
@@ -6423,12 +6419,15 @@ def hist(self, x, bins=None, range=None, density=None, weights=None,
         cumulative : bool, optional
             If ``True``, then a histogram is computed where each bin gives the
             counts in that bin plus all bins for smaller values. The last bin
-            gives the total number of datapoints. If *normed* or *density*
-            is also ``True`` then the histogram is normalized such that the
-            last bin equals 1. If *cumulative* evaluates to less than 0
-            (e.g., -1), the direction of accumulation is reversed.
-            In this case, if *normed* and/or *density* is also ``True``, then
-            the histogram is normalized such that the first bin equals 1.
+            gives the total number of datapoints.
+
+            If *density* is also ``True`` then the histogram is normalized such
+            that the last bin equals 1.
+
+            If *cumulative* is to less than 0 (e.g., -1), the direction of
```

Comment:
```suggestion
            If *cumulative* is a number less than 0 (e.g., -1), the direction of
```
That should be mentioned in the type. Just `bool` is plain confusing. Maybe we could also deprecate numbers in favor of "reversed", but that's for another PR.

Reply:
I now documented the type as `bool or -1` which is technically too restrictive but more informative in practice, I think.